aten/src/ATen/native/native_functions.yaml

```diff
@@ -3096,6 +3096,9 @@
 - func: nonzero(Tensor self) -> Tensor
   variants: method, function
 
+- func: nonzero_tuple(Tensor self) -> Tensor[]
```

Comment:
it's a little strange to call this nonzero_tuple when we don't return a tuple (in C++), and the behavior isn't just "return a tuple" (because of numpy compatbility concerns.  Maybe just call this nonzero_numpy?